lc0_main.go

```diff
@@ -238,7 +238,7 @@ func convertMovesToPGN(moves []string, result string) string {
 		b = []byte(strings.TrimRight(b_str, "*") + to_append)
 	}
 	game2.UnmarshalText(b)
-	return game2.String()
+	return game2.String() + " {TDS: " + strconv.Itoa(start_ply_count) + "}"
```

Comment:
Maybe instead of adding comment in the end, insert `{Book ends here.}` right after last "book" move?
And what does "TDS" stand for?

Reply:
TDS is 'training data starts' (0 indexed for maximal confusion :P).  I also considered BL for book length.

Inserting into the middle was my original plan, but since the golang chess library doesn't support pgn comments, I would have to manually calculate the right place to insert into the string.

Reply:
Changed to OL for opening length.